app/donu/Main.hs

```diff
@@ -180,39 +156,41 @@ handleRequest r =
     QueryModels QueryModelsCommand {..} ->
       succeed <$> liftIO (queryModels queryParameters)
 
-    ExecuteExposureCode (ExecuteExposureCodeCommand code) ->
-      case lexExposure (Text.unpack code) >>= parseExposureStmts of
-        Left err ->
-          pure $ FailureResult $ Text.pack err
-        Right stmts ->
-          stepExposureSession stmts
-
-    ResetExposureState _ ->
-      do Snap.with exposureSessions $
-           putExposureSessionState Exposure.initialEnv
-         succeed' ()
+    -- ExecuteExposureCode (ExecuteExposureCodeCommand code) ->
```

Comment:
There's various commented-out code in this file, such as these lines. Is there any reason to keep these around as comments?

Reply:
Nope, I just didn't actually include deleting those in a commit for whatever reason. Thanks.